moz_sql_parser/sql_parser.py

```diff
@@ -371,9 +374,104 @@ def to_string(instring, tokensStart, retTokens):
 ).addParseAction(to_union_call)
 
 
-SQLParser = selectStmt
+def to_create_json_call(instring, tokensStart, retTokens):
+    # ARRANGE INTO {op: params} FORMAT
+    tok = retTokens
+    name = tok[0].lower()
+
+    types = tok.types
+    option = tok.option
+    if not types:
+        types = tok[1]
+    if not option:
+        option = tok[2:4]
+        op = options_call(option)
+    if len(tok) == 2:
+        return {"name" : name, "type": types}
+    elif len(tok) == 3 or 4:
+        return {"name" : name, "type": types, "option" : op}
+    else:
+        return tok
+
+def options_call(option):
+    op = str(option)
+    if "not" and "null" in op:
+        return "not null"
+    elif "primary" in op:
+        return "primary key"
+    elif "foreign" in op:
+        return "foreign key"
+    elif "check" in op:
+        return "check"
+    elif "default" in op:
+        return op
+    elif "index" in op:
+        return "index"
+    else:
+        return None
+
+
+def to_create_table_call(instring, tokensStart, retTokens):
+    tok = retTokens
+
+    if tok:
+        return {"create table" : tok.asList()}
+
+def to_table_name_call(instring, tokensStart, retTokens):
+    tok = retTokens
+
+    if tok[0]:
+        return {"name" : tok[0] }
+
+
+def to_columns_call(instring, tokensStart, retTokens):
+    tok = retTokens
+
+    return {"columns" : tok.asList()}
+
+
+createStmt = Forward()
+
+column_name = ident.copy().setName("column_name").setDebugActions(*debug)
+
+column_size = Group(
+                Literal('(').suppress().setDebugActions(*debug) +
+                intNum.copy().setName("size").setDebugActions(*debug) +
```

Comment:
If I remember correctly, every pattern has two names.  The name used for parsing `setName("size")` and the name used for the output token `("size")`.  You should not need to `copy()`, rather 

    intNum("size").setDebugActions(*debug) +

Reply:
yes, it can handle negative numbers.  Fix that if you want.

Reply:
Because of the user never puts negative integers in the `column_size`. So I replaced it with `intNum("size").setDebugActions(*debug)` and not fixing for positive integers.